Tasks/BashV3/bash.ts

```diff
@@ -79,7 +79,7 @@ async function run() {
             if (old_source_behavior) {
                 contents = `. '${targetFilePath.replace(/'/g, "'\\''")}' ${input_arguments}`.trim();
             } else {
-                contents = `bash '${targetFilePath.replace(/'/g, "'\\''")}' ${input_arguments}`.trim();
+                contents = `exec bash '${targetFilePath.replace(/'/g, "'\\''")}' ${input_arguments}`.trim();
```

Comment:
Why do we need to exec here? What's the purpose of this change?

Reply:
This change allows to the script to catch signals that are passed from toolRunner. Exec allows to user script file to be executed as subprocess of our Bash and CmdLine tasks. As we are wraping input scripts into temp script file, execution of user's scripts is starting as subprocess of the process where this temp script is executing and as a result user's script couldn't catch cancellation signal from task.